config/locales/views/schools/schools.yml

```diff
@@ -388,6 +388,8 @@ en:
       form:
         email: Email
         name: Name
+        preferred_locale: Preferred language
+        preferred_locale_hint: This will be the language website and email content will be in for this user
```

Comment:
"Set the user's preferred language for using the website and receiving emails"

Reply:
👍 done